open-company/open-company-web
# Topic edit

card: https://trello.com/c/FiGpHNm4

This use tinyMCE as main editor for rich content fields.

To test:
- open the dashboard of a company
- expand a topic
- tap on the topic body: you should see the edit button at the bottom of the viewport (if it has no body the button should appear at the previous step)
- edit the topic: try using the new tinyMCE on mobile too if you like (i didn't test it no android as i don't have my tablet with me but i tested it on the iOS simulator and on my iPhone)
- save (or cancel)
- check that the changes are reflected
